test/smtchecker.ts

```diff
@@ -0,0 +1,50 @@
+import * as tape from 'tape'
+import smtchecker from '../smtchecker'
+
+tape('SMTChecker', function (t) {
+  t.test('smoke test with no axuiliaryInputRequested', function (st) {
+    var input = {};
```

Comment:
Why do these need to be changed back to `var`?

Reply:
these are const.